data/sitewide_alert.yml

```
@@ -0,0 +1,20 @@
+# Site Wide Alert setting that is displayed at the top of the website
+# https://designsystem.digital.gov/components/site-alert/
+#
+# display_alert: hide/show the sitewide alert on the site
+# is_emergency: true will display with orange emergency colors else it will display as blue info box
+# has_icon: hide/show icon
+# is_slim: sets a thinner banner height
+# header: bold text that appears next the icon at top of alert box
+# alerts: text field that supports markdown, will display a bulleted list if multiple, otherwise displays as standard text block
+
+data:
+  display_alert: false
+  is_emergency: false
+  has_icon: true
+  is_slim: false
+  header: "Guidance for U.S. government websites and social media for:"
+  alerts:
+    - "Hawaii wildfires: [digital.gov/hawaii-wildfires](/hawaii-wildfires)"
+    # - "Hawaii wildfires: [digital.gov/hawaii-wildfires](/hawaii-wildfires)"
```

Could use a unique example, like the recent hurricane notice.

```suggestion
    # - "Hurricane idalia: [digital.gov/hurricane-idalia](/hurricane-idalia)"
```

---

I will remove this, it's not really needed.